jaeger-core/src/main/java/io/jaegertracing/internal/JaegerObjectFactory.java

```diff
@@ -0,0 +1,46 @@
+package io.jaegertracing.internal;
+
+import io.jaegertracing.internal.clock.Clock;
+import io.jaegertracing.internal.metrics.Metrics;
+import io.jaegertracing.spi.BaggageRestrictionManager;
+import io.jaegertracing.spi.Reporter;
+import io.jaegertracing.spi.Sampler;
+import io.opentracing.ScopeManager;
+
+import java.util.List;
+import java.util.Map;
+
+public class JaegerObjectFactory {
+  public JaegerSpan createSpan(
+      JaegerTracer tracer,
+      String operationName,
+      JaegerSpanContext context,
+      long startTimeMicroseconds,
+      long startTimeNanoTicks,
+      boolean computeDurationViaNanoTicks,
+      Map<String, Object> tags,
+      List<Reference> references) {
+    return new JaegerSpan(
+        tracer,
+        operationName,
+        context,
+        startTimeMicroseconds,
+        startTimeNanoTicks,
+        computeDurationViaNanoTicks,
+        tags,
+        references);
+  }
+
+  public JaegerSpanContext createSpanContext(long traceId,
```

Comment:
formatting inconsistent with the previous method

Reply:
Ya will fix formatting after main code is good to go.